src/Sylius/Bundle/InstallerBundle/Behat/CliContext.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\InstallerBundle\Behat;
+
+use Sylius\Bundle\InstallerBundle\Command\SetupCommand;
+use Sylius\Bundle\ResourceBundle\Behat\DefaultContext;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\HttpKernel\Kernel;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+require_once __DIR__.'../../../../../../vendor/phpunit/phpunit/src/Framework/Assert/Functions.php';
+
+/**
+ * @author Magdalena Banasiak <magdalena.banasiak@lakion.com>
+ */
+class CliContext extends DefaultContext
+{
+    /**
+     * @var Application
+     */
+    private $application;
+
+    /**
+     * @var CommandTester
+     */
+    private $tester;
+
+    public function __construct()
+    {
+        parent::__construct();
+        $this->application = new Application($this->getKernel());
```

Comment:
`$this->getKernel()` returns null, anybody can suggest me, why? Any solutions? :D